package.json

```diff
@@ -14,6 +14,6 @@
   "license": "Apache-2.0",
   "dependencies": {
     "cql-exec-fhir": "^2.1.4",
-    "cql-execution": "^2.4.0"
+    "cql-execution": "^3.0.1"
   }
 }
```

Comment:
Let's bump the version of cql-worker on line 3 so that we can publish a new package to [npm](https://www.npmjs.com/package/cql-worker/v/1.1.6?activeTab=versions). Do you think this is a patch change or a minor change?

Reply:
I will create another PR to upgrade version to 1.2.0